### ISSUE TITLE
Duplicate entries appearing

### ISSUE THREAD

**Issue by ricotheque** [toggl/mobileapp#1493]
/case/643749: Duplicate entries appear after stopping timer.

/case/643152: Duplicate entries appear on sync. Attempting to delete duplicate entries crashes the app.

/case/643763: Duplicate (and even triplicate!) entries appear "after I adjust the duration wheel". Most likely this means duplicates appear after editing an entry.

/case/643764: Duplicates appear after entry is added, when the timer is started, or after the timer is stopped. Confirms that duplicate entries aren't being synced.

**Comment by amulware** [comment]
Any reliable steps to reproduce would be great.

**Comment by ricotheque** [comment]
Yep, waiting on recorded videos from some cases.

**Comment by amulware** [comment]
Added to the current milestone in case we can fix it quickly. If we cannot reproduce it though, I don't want this to be the focus for the whole week.

**Comment by ricotheque** [comment]
/case/643749 (iOS 11.2.5) has just forwarded a video of a duplicate just appearing, with no action taken by the user.

I'll forward more details from others as I receive them.

**Comment by nicktoggl** [comment]
One user has told us that if he goes to settings or closes the app and reopens it, the duplicates are gone. Additionally he checked and knew that the duplicates weren't getting put on our server.

**Comment by amulware** [comment]
This has likely been resolved by the PR above and will be release in `hotfix 1.3.2`.

If this but is still reported with version 1.3.2 or higher, please reopen/create a new issue.
